src/tools/rados/rados.cc

```diff
@@ -2589,6 +2590,20 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
     } else {
       ret = 0;
     }
+  } else if (strcmp(nargs[0], "clearomap") == 0) {
+    if (!pool_name || nargs.size() < 2) {
+      usage_exit();
+    }
+
+    string oid(nargs[1]);
```

Comment:
How about something like this?
```
for (unsigned i = 1; i < nargs.size(); ++i) {
  std::string oid(nargs[i]);
  ret = io_ctx.omap_clear(oid);
  if (ret < 0)
    cerr << "say somethign..." << cpp_strerror(ret) << '\n';
}
```
So that we can do this

`./bin/rados -p test_t clearomap a b c d e `

Reply:
added a commit to do this.